Makefile

```diff
@@ -186,17 +188,30 @@ lint-quick:
 	$(VIRTUAL_ENV)/bin/python manage.py makemigrations --dry-run --check --noinput || EXIT_STATUS=$$?; \
 	exit $${EXIT_STATUS}
 
-.PHONY: lint-fix
-lint-fix:
+.PHONY: lint-js-fix
+lint-js-fix:
 	EXIT_STATUS=0; \
-	npm run lint-fix ||  EXIT_STATUS=$$?; \
+	npm run lint-fix || EXIT_STATUS=$$?; \
+	exit $${EXIT_STATUS}
+
+# Use with caution, the automatic fixing might produce bad results
+.PHONY: lint-html-fix
+lint-html-fix:
+	EXIT_STATUS=0; \
+	$(VIRTUAL_ENV)/bin/djlint $(ARGUMENTS) --reformat --profile=django || EXIT_STATUS=$$?; \
+	exit $${EXIT_STATUS}
+
+.PHONY: lint-html-files
+lint-html-files:
+	EXIT_STATUS=0; \
+	$(VIRTUAL_ENV)/bin/djlint $(ARGUMENTS) --profile=django --ignore=H030,H031 || EXIT_STATUS=$$?; \
 	exit $${EXIT_STATUS}
 
 .PHONY: lint-python-files
 lint-python-files:
 	EXIT_STATUS=0; \
 	$(VIRTUAL_ENV)/bin/black $(ARGUMENTS) || EXIT_STATUS=$$?; \
-	$(VIRTUAL_ENV)/bin/isort --diff -c $(ARGUMENTS) --filter-files || EXIT_STATUS=$$?; \
+	$(VIRTUAL_ENV)/bin/isort $(ARGUMENTS) --filter-files || EXIT_STATUS=$$?; \
```

Comment:
I removed `--diff -c` so it tries to autofix the broken ones

Reply:
@goapunk shall we adjust this in our other projects too?

Reply:
probably yes